pkg/daemon/update.go

```diff
@@ -325,6 +325,22 @@ func (dn *Daemon) updateFIPS(current, desired *mcfgv1.MachineConfig) error {
 	if current.Spec.FIPS == desired.Spec.FIPS {
 		return nil
 	}
+
+	cmd := exec.Command(fipsCommand)
```

Comment:
Seems odd to shell out to `cat` as opposed to reading the file directly...

Reply:
Yes sorry let me fix that, I just wanted to check if this flow was what people wanted